dashboard/client/src/components/FunctionTableItem/FunctionTableItem.jsx

```diff
@@ -17,6 +17,69 @@ const genRepoUrl = ({ gitOwner, gitRepoURL }) => (
   `${gitRepoURL}/commits/master`
 );
 
+const genOwnerInitials = (owner) => (
+  (owner.split(/[-_]+/).length > 1) ?
+    owner.split(/[-_]+/)[0].substring(0, 1).concat(owner.split(/[-_]+/)[1].substring(0, 1)) :
+      owner.split(/[-_]+/)[0].substring(0, 2)
+
+);
+
+const stringToColour = (str) => {
+  const hash = [...str].reduce((acc, curr) => curr.charCodeAt(0) + ((acc << 5) - acc), 0);
+
+  let colour = '#';
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
Can we comment more about what's happening here? Is any of this code from StackOverflow or a blog post? (we could link to the source)

Reply:
I will add the link: https://stackoverflow.com/questions/3426404/create-a-hexadecimal-colour-based-on-a-string-with-javascript

Reply:
I'd definitely add a link to SO here